packages/examples/sample_app/lib/Views/ImageUploader.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'dart:io';
+import 'package:file_picker/file_picker.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'package:amplify_storage_s3/amplify_storage_s3.dart';
+
+class ImageUploader extends StatelessWidget {
+  void _upload(BuildContext context) async {
+    try {
+      print('In upload');
+      // Uploading the file with options
+      File local = await FilePicker.getFile(type: FileType.image);
+      local.existsSync();
+      final key = new DateTime.now().toString();
+      Map<String, String> metadata = <String, String>{};
```

Comment:
Is the metadata really necessary for the sample app?

Reply:
I think it's good to include, it's an example on how to use it for the developer.